### PR TITLE
Disabled button

### LINKED ISSUE #183: Add support for disabled prop on Button
When we write the following code:

```tsx
<Button disabled>Submit</Button>
```

It should look like this:
image

- Add storybook example

### PR DESCRIPTION
**Pull request by nuha90**
# Description

Add support for disabled prop on Button
Fixes #183 

# Checklist

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by tpetrina — CHANGES_REQUESTED** [review]
I think you misunderstood this task. You need to add the `disabled` property to the existing `Button` component and style it with `::disabled` pseudo selector.

**Comment by tpetrina** [comment]
@nuha90 https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled

File: src/client/components/Button/Button.js [hunk 1]
```diff
@@ -8,10 +8,11 @@ export const Button = (props) => {
     <button
       className={`button ${
         props.appearance === 'danger' ? 'button-danger' : ''
-      }`}
+      }${props.disabled === 'disabled' ? 'disabled-button' : ''}`}
```
Comment (tpetrina) [inline src/client/components/Button/Button.js:11]:
```suggestion
      } ${props.disabled === 'disabled' ? 'disabled-button' : ''}`}
```
You need a space here.

**Review by tpetrina — APPROVED** [review]
Pretty good stuff!

**Review by tpetrina — APPROVED** [review]

**Review by 1620prabn — APPROVED** [review]

**Pull request merged**